client/vscode/CONTRIBUTING.md

```diff
@@ -201,16 +201,16 @@ The release process for the VS Code Extension for Sourcegraph is currently autom
 #### Release Steps
 
 1. Create a new branch when the main branch is ready for release, or use your current working branch if it is ready for release
-2. Run `yarn workspace @sourcegraph/vscode run release:$RELEASE_TYPE` in the root directory
+2. Run `pnpm --filter @sourcegraph/vscode run release:$RELEASE_TYPE` in the root directory
```

Comment:
Is `--filter` clearer than using `-C`?

Reply:
I'm not sure, I think I got `--filter` from a random google search saying to use it in place of `yarn workspace`, but maybe a "filter" is too generic?

It seems like it would be `-C client/vscode` might be more what we want, to run in a specific folder and only in that folder. Where I think `--filter` will run in any folder that matches.

However `--filter @sourcegraph/vscode` seems to work which I assume uses the package name (`name` field from package.json`. This is probably the closest or equivelent to `yarn workspace`?

Reply:
Looking through the docs, it looks like `-C` isn't really documented. I think you're right and `--filter` is the way to go.